controllers/user.js

```diff
@@ -111,14 +111,19 @@ async function getAllEmailsAndBreaches(user, allBreaches) {
 
 
 async function getDashboard(req, res) {
+  _requireSessionUser(req);
   const allBreaches = req.app.locals.breaches;
   const user = req.session.user;
   const { verifiedEmails, unverifiedEmails } = await getAllEmailsAndBreaches(user, allBreaches);
 
+  const newBreachesFound = allBreaches.filter(breach => breach.AddedDate >= user.breaches_last_shown);
```

Comment:
Just so I'm clear, is the intent here to display all new HIBP breaches since the user's last visit, and not just new breaches that the user was involved in?

Reply:
Should be only new breaches that the user was involved in so we'll just need to loop through their `foundBreaches` instead of `allBreaches`.